newrelic.yml

```diff
@@ -395,6 +395,13 @@ common: &default_settings
 # May be one of [auto|prepend|chain|disabled].
 # instrumentation.typhoeus: auto
 
+# Controls auto-instrumentation of the Thread class at start up to allow the agent to correctly nest spans inside of an asyncronous transaction.
+# May be one of [auto|prepend|chain|disabled].
+# instrumentation.thread: auto
+
+# Controls auto-instrumentation of the Thread class at start up to automatically add tracing to all Threads created in the application.
+# instrumentation.thread.tracing: true
```

Comment:
I think the boolean value here should match what is in `default_source.rb` (false), perhaps with a note that explicitly says "set to \`true\` to enable" or whatnot.